src/main/java/org/jboss/msc/service/ServiceContainerImpl.java

```diff
@@ -844,7 +844,7 @@ public void uncaughtException(final Thread t, final Throwable e) {
             ServiceLogger.ROOT.uncaughtException(e, t);
         }
     };
-    private static final ThreadPoolExecutor.CallerRunsPolicy POLICY = new ThreadPoolExecutor.CallerRunsPolicy();
+    private static final ThreadPoolExecutor.AbortPolicy POLICY = new ThreadPoolExecutor.AbortPolicy();
```

Comment:
I'll remove this commit too.
There is a wrong test in MSC test case
MultipleRemoveListenerTestCase.testDoneAfterRemoval()
that fails sporadically. This test expects it is possible to
schedule new tasks after container shutdown.
The test will be eliminated.